### DISCUSSION TITLE
What exactly is a "lazy line"

### DISCUSSION THREAD

**Discussion by Ahacad** [micromark/micromark#72] [Q&A] [answered]
Hi I'm trying to write a syntax extension for micromark, and I've encountered `self.parser.lazy` which points to the concept of the lazy line, I've read some of the comments in `packages/micromark/dev/lib/initialize/document.js` but still have questions for what exactly is a lazy line (and what is a nonlazy line), could you give some examples and definitions?

**Comment by ChristianMurphy** [comment, marked as answer]
definition and examples from the commonmark specification: https://spec.commonmark.org/0.30/#lazy-continuation-line and https://spec.commonmark.org/0.30/#block-quote-marker
